tests/test_box_Box.py

```diff
@@ -485,6 +485,28 @@ def test_from_box(self):
         box7 = freud.box.Box.from_matrix(box.to_matrix())
         assert np.isclose(box.to_matrix(), box7.to_matrix()).all()
 
+    def test_standard_orthogonal_box(self):
+        box = freud.box.Box.from_box((1, 2, 3, 0, 0, 0))
+        Lx, Ly, Lz, alpha, beta, gamma = box.to_box_lengths_and_angles()
+        npt.assert_allclose(
+            (Lx, Ly, Lz, alpha, beta, gamma), (1, 2, 3, np.pi / 2, np.pi / 2, np.pi / 2)
+        )
+
```

Comment:
I think a great test to add would be to start with 3 random lengths and 3 random angles, then call `from_box_lengths_and_angles` to get a box, then call `to_box_lengths_and_angles` on that box and assert that you recover the original random lengths and angles.